src/pyronn_torch/conebeam.py

```diff
@@ -143,18 +143,18 @@ def from_conrad_config(cls):
             pyconrad.config.get_geometry().getPixelDimensionX(),
             pyconrad.config.get_geometry().getPixelDimensionY(),
         ]
-        projection_origin = [
-            pyconrad.config.get_geometry().getDetectorOffsetU(),
-            pyconrad.config.get_geometry().getDetectorOffsetV(),
-        ]
+        # projection_origin = [
```

Comment:
but why out-comment the projection origin?

Reply:
sorry, that was experimental because it's not used in the constructor, but I didn't mean to push to master. I removed the changes.